docs/user-guide/address-security-requirements.md

```diff
@@ -0,0 +1,88 @@
+| API Mediation Layer certificate mapping | [Configuring main Zowe server to use client certificate identity mapping](../user-guide/configure-zos-system.md/#configure-main-zowe-server-to-use-client-certificate-identity-mapping) |
+| API Mediation Layer identity mapping | [Configuring main Zowe server to use distributed identity mapping](../user-guide/configure-zos-system.md/#configure-main-zowe-server-to-use-distributed-identity-mapping) |
+| API Mediation Layer Identity Tokens (IDT) | [Configuring signed SAF Identity tokens (IDT)](../user-guide/configure-zos-system.md/#configure-signed-saf-identity-tokens-idt) |
+| Cross memory server (ZIS) | [Configuring the cross memory server for SAF](../user-guide/configure-zos-system.md/#configure-the-cross-memory-server-for-saf)<br />and<br />[Configuring cross memory server load module](../user-guide/configure-xmem-server.md/#load-module)<br />and<br />[Configuring cross-memory server SAF configuration](../user-guide/configure-xmem-server.md/#saf-configuration) |
+
+## Assign security permissions of users
+
+As a security administrator, assign users and the ZWEADMIN security group permissions required to perform specific tasks. 
+
+### Overview of user categories and roles
+
+Specific user IDs with sufficient permissions are required to run or access Zowe. 
+Your organization's security administrator is responsible to assign the following user IDs during Zowe z/OS component configuration.
+
+The following user IDs can run Zowe: 
+
+* **[ZWESVUSR](#zwesvusr)**  
+This user runs most of the Zowe core components.
+* **[ZWESIUSR](#zwesiusr)**  
+This user runs the cross memory server (ZIS)
+
+The security administrator also assigns permissions to the security group [ZWEADMIN](#zweadmin), as well as permissions to individual Zowe users.
+
+### ZWESVUSR
+
+This is a started task ID for `ZWESLSTC`.  
+
+The task starts a USS environment using `BPXBATSL` that executes the core Zowe Desktop (ZLUX) node.js server, the Java API Mediation Layer, and the Z Secure Services C component.  To work with USS, the user ID `ZWESVUSR` must have a valid OMVS segment.  
+
+### ZWESIUSR
+
+This is a started task ID used to run the PROCLIB `ZWESISTC` that launches the [cross memory server (ZIS)](./configure-xmem-server.md). This started task ID must have a valid OMVS segment.
+
+### ZWEADMIN
+
+ZWEADMIN is a group that `ZWESVUSR` and `ZWESIUSR` should belong to. This group must have a valid OMVS segment.  
+
+### zowe_user
+
+If z/OSMF is used for authentication and serving REST APIs for Zowe CLI and Zowe Explorer users, the TSO user ID for end users must belong to one or both of the groups `IZUUSER` or `IZUADMIN`.
+
+## Security Permissions Reference Table 
```

Comment:
I am not certain but I think this table describes permissions only for ZWESVUSR.